src/components/Selection/selection.spec.jsx

```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types'
 import { shallow } from 'enzyme'
 import selectionContainer from './selectionContainer'
 import AppLike from '../../tests/Applike'
-const DummyComponent = ({ title = '' }) => <span title={title} />
-const DummyComponentWithSelection = selectionContainer(DummyComponent)
 import configureStore from '../../store/configureStore'
 import getCozyClient from '../../tests/client'
 
+const DummyComponent = ({ title = '' }) => <span title={title} />
 DummyComponent.propTypes = {
   title: PropTypes.string
 }
+const DummyComponentWithSelection = selectionContainer(DummyComponent)
+
+// Uses a different version of react-redux
+// to prevent Enzyme's incompatibility with actual react-redux version
+jest.mock('react-redux', () => require('react-redux-test'))
```

Comment:
Can you link to an issue or something?

Reply:
What if you use mount instead of shallow? 

I really don't like having 2 versions of react-redux, the test doesn't test the code used in our app...

Reply:
Even if the hack is scoped, the test is not relevant anymore. We should create an issue on Contacts to rewrite this test with react-testing-lib. Since one of the next task will be to work on list's filter, maybe we will have to work on the selection part and fix this issue

Reply:
issue : https://github.com/cozy/cozy-contacts/issues/665